src/OpenGraph.php

```diff
@@ -7,9 +7,9 @@
 
 class OpenGraph
 {
-    public function fetch($url, $allMeta = null, $lang = null, $options = LIBXML_NOWARNING | LIBXML_NOERROR)
+    public function fetch($url, $allMeta = null, $lang = null, $userAgent = 'Curl', $options = LIBXML_NOWARNING | LIBXML_NOERROR)
```

Comment:
@huulbaek Can we make the $userAgent fourth parameter so that it's backward compatible? Consider if users are using this function like this.
`$data = OpenGraph::fetch($url, $allMeta, $language, $options);  
`

making $userAgent as the fourth parameter may introduce errors. Consider doing this instead.

`$data = OpenGraph::fetch($url, $allMeta, $language, $options, $userAgent);  
`

Or let me know the reasoning behind moving the $userAgent before $options.

Reply:
Yes, that's fine by me. I didn't consider backwards compatibility – I mostly considered how often it would be used.